### PR TITLE
WB-98: docs(): Update README.md

### PR DESCRIPTION
**Pull request by jandrade**
## Summary

Updated README.md to show a `getting started` section that includes short installation instructions and adds the URL to see the docs locally. Also, added a link to the confluence page for a more detailed explanation on the process.

Output: https://github.com/Khan/wonder-blocks/blob/internal/readme-docs/README.md

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 640ba6ff76445859c06fe4fd676c04ad2bf8a489

https://deploy-preview-471--wonder-blocks.netlify.com

File: README.md [hunk 1]
```diff
@@ -2,10 +2,42 @@
...
+
+#### `yarn install`
+
+Installs project dependencies
+
+#### `yarn start`
+
+Runs the docs in dev mode
+
+Now you can open http://localhost:6060 to view the docs. This page will
```
Comment (somewhatabstract) [inline README.md]: Yes! Thank you!

File: README.md [hunk 2]
```diff
@@ -2,10 +2,42 @@
...
+automatically update as you make changes to components.
+
 ## Contributing
```
Comment (somewhatabstract) [inline README.md]: I think we should add an explicit statement about accepting external contributions that makes it clear if we do or do not, and if we do, under what circumstances (i.e. what types of changes).

Comment (jandrade) [reply README.md]: That's a great suggestion, I'll add that shortly.

**Pull request merged**